x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/entsearch/engine/EngineIndexService.java

```diff
@@ -215,6 +284,7 @@ public void putEngine(Engine engine, ActionListener<IndexResponse> listener) {
      */
     public void deleteEngine(String engineName, ActionListener<DeleteResponse> listener) {
         try {
+            // TODO Delete alias when Engine is deleted
```

Comment:
I've added this to the acceptance criteria of the followup issue, this comment is no longer needed. Thank you!